backend/data/src/main/kotlin/io/tolgee/batch/BatchJobChunkExecutionQueue.kt

```diff
@@ -63,14 +63,21 @@ class BatchJobChunkExecutionQueue(
         from BatchJobChunkExecution bjce
         join bjce.batchJob bk
         where bjce.status = :executionStatus
-        order by bjce.createdAt asc, bjce.executeAfter asc, bjce.id asc
+        order by 
+          case when bk.status = 'RUNNING' then 0 else 1 end,
+          bjce.createdAt asc, 
+          bjce.executeAfter asc, 
+          bjce.id asc
         """.trimIndent(),
         BatchJobChunkExecutionDto::class.java,
       ).setParameter("executionStatus", BatchJobChunkExecutionStatus.PENDING)
         .setHint(
           "jakarta.persistence.lock.timeout",
           LockOptions.SKIP_LOCKED,
-        ).resultList
+        )
+        // Limit to get pending batches faster
+        .setMaxResults(1_000)
```

Comment:
it limits how many chunks are added to the queue for processing every minute. I think such thing should be parametrized. I would put it to the `io.tolgee.configuration.tolgee.BatchProperties`

Reply:
Oh right sorry, you mentioned it earlier.

Will fix.